src/parallax/first/index.css

```diff
@@ -0,0 +1,55 @@
+.parallax1 {
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  grid-template-rows: repeat(12, 1fr);
+  align-items: center;
+  transform-style: preserve-3d;
+  min-height: calc(100vh - 90px);
+}
+
+.left {
+  transform: translateZ(-40px) scale(1.13);
+  transform-origin: right;
+  grid-area: 2 / 1 / 11 / 7;
+  transform-origin: right;
+}
+
+.left h1 {
+  font-size: 4.8rem;
+  font-weight: 500;
+  text-align: right;
+  letter-spacing: 1px;
+  line-height: 1.1;
+}
+
+.right {
+  grid-area: 2 / 6 / 11 / 13;
+  height: 400px;
+  position: relative;
+  transform: translateZ(-120px) scale(1.4);
+  transform-origin: left;
+}
+
+.p1-background-image {
+  opacity: 0.7;
+  position: absolute;
+  height: 400px;
+  width: 470px;
+  filter: grayscale(70%);
+  background-image: url("./background.png");
+  background-size: cover;
+  background-position: center center;
+}
+
+.p1-front-image {
+  width: 130px;
+  transform: translateZ(-100px) scale(1.33);
+  transform-origin: left;
+  position: absolute;
+  left: calc(41.6% + 233px);
+  top: calc(62% - 150px);
```

Comment:
question: explain me this, why position absolute with left and top? Have you noticed that this doesn't look very good (the same) in large screens vs medium screens? (the tricky with the zoom + and -

Reply:
More important than knowing better solution it's understanding what was wrong with the initial solution. That's why I asked you to write it down (even after watching the looms I shared eheh). Here's a [happy mem](https://github.com/alexodan/parallax-demo/assets/14869087/1c1e8b87-c5a1-4f3b-8172-470203856382) :p